cypress/support/admin.ts

```diff
@@ -1,10 +1,25 @@
+import { GetSettingsQuery, PrepareDbMutation } from '../../src/generated/sdk';
 import { getE2EApi } from './utils';
 
-const resetDB = (includeSeeds = false) => {
+const resetDB = (
+  includeSeeds = false
+): Cypress.Chainable<PrepareDbMutation> => {
   const api = getE2EApi();
   const request = api.prepareDB({ includeSeeds });
 
-  cy.wrap(request);
+  return cy.wrap(request);
+};
+
+const getAndStoreAppSettings = (): Cypress.Chainable<GetSettingsQuery> => {
```

Comment:
If this is something that needs to be run always maybe it is worth looking into support file before hook
https://docs.cypress.io/guides/references/configuration#Folders-Files
https://stackoverflow.com/a/48153074/813773

Reply:
Will definitely look into that option.